Developers.md

```diff
@@ -21,6 +21,11 @@ If you would like to `bazel` to use a local installation of drake, you can set
 the `DRAKE_INSTALL_DIR` environment variable. Otherwise it will look in
 `/opt/drake`.
 
+You might need to install [tidy](https://www.html-tidy.org/):
+```bash
+sudo apt install tidy
+```
```

Comment:
i guess you didn't run `setup/.../install_prereqs.sh` in this repo?

Reply:
I did not. The only instructions I found after a quick look were in `Developers.md`. It might be worth adding some instructions to the README. What do you think?

Reply:
sounds good to me.

Reply:
Let's add any useful information here then.

So far, I know the following:
- Run `setup/.../install_prereqs.sh`
- See `Developers.md` for developer-specific instructions
   - Here we should probably add that requirements need to be added to all `...-requirements.txt` files and then generated with `htmlbook/requirements.sh`
   
 Is there anything that I'm missing?